raft-tech/TANF-app
# Gitattributes update for silencing "modified" pdfs

### Summary of Changes
_Provide a brief summary of changes_
_Addresses issue #1235
_Acceptance criteria as stated in the issue_
Prevent pdfs showing as modified.

### How to Test
_List the steps to test the PR_
On a clean checkout of raft-tdp-main, run `git status`
You may see the modified files as per the ticket's screenshot.

Switch to this branch hotfix/1235-pdfs-showing-modified and again run `git status` to see that pdfs do not show as modified.
